x/axelarnet/keeper/msg_server_test.go

```diff
@@ -124,6 +119,12 @@ func TestHandleMsgConfirmDeposit(t *testing.T) {
 			EnqueueForTransferFunc: func(sdk.Context, nexus.CrossChainAddress, sdk.Coin, sdk.Dec) (nexus.TransferID, error) {
 				return nexus.TransferID(mathRand.Uint64()), nil
 			},
+			GetChainByNativeAssetFunc: func(ctx sdk.Context, denom string) (nexus.Chain, bool) {
+				if denom == exported.Uaxl {
+					return exported.Axelarnet, true
+				}
+				return nexus.Chain{}, true
```

Comment:
shouldn't this be:

```suggestion
				return nexus.Chain{}, false
```

?